components/02-molecules/registration/RequestToRegisterComponent.tsx

```diff
@@ -42,10 +46,16 @@ export const RequestToRegisterComponent = ({
       throw new Error("Impossible to register a name without a name");
     }
 
+    let resolverAddress =
+      nameRegistrationData.ensResolver === EnsResolver.Custom &&
+      nameRegistrationData.customResolverAddress
+        ? nameRegistrationData.customResolverAddress
+        : ensResolverAddress[nameRegistrationData.ensResolver];
+
```

Comment:
Does it correctly fail if no Custom Resolver address is informed?

Reply:
You can't proceed the registration process if no Custom resolver is informed